config/crd/bases/toolchain.dev.openshift.com_toolchainconfigs.yaml

```diff
@@ -272,6 +285,12 @@ spec:
                             description: Defines all secrets related to the registration
                               service verification configuration
                             properties:
+                              awsAccessKeyID:
+                                description: AWSAccessKeyId
+                                type: string
+                              awsSecretAccessKey:
+                                description: AWSSecretAccessKey
+                                type: string
```

Comment:
Don't forget to regenerate the description if you add some comments as proposed in this comment: https://github.com/codeready-toolchain/api/pull/321#discussion_r973924604

Reply:
Done